core-test/src/main/resources/storage/v1/volume.json

```diff
@@ -7,7 +7,8 @@
                 "device": "/dev/vdd",
                 "server_id": "eaa6a54d-35c1-40ce-831d-bb61f991e1a9",
                 "id": "8a9287b7-4f4d-4213-8d75-63470f19f27c",
-                "volume_id": "8a9287b7-4f4d-4213-8d75-63470f19f27c"
+                "volume_id": "8a9287b7-4f4d-4213-8d75-63470f19f27c",
+                "attachment_id": "f0dcbc70-52ee-11ea-8d77-2e728ce88125"
```

Comment:
Pardon my silly question, but have you verified the response format against a living instance?